core/state/trie_prefetcher.go

```diff
@@ -49,49 +37,114 @@ var (
 type triePrefetcher struct {
 	db       Database                    // Database to fetch trie nodes through
 	root     common.Hash                 // Root hash of theaccount trie for metrics
+	fetches  map[common.Hash]Trie        // Partially or fully fetcher tries
```

Comment:
it's quite easy to misread `fetches` and `fetchers`, when skimming through the code